src/fidesops/service/privacy_request/request_runner_service.py

```diff
@@ -190,7 +204,6 @@ def run_privacy_request(
                 after_webhook_id=from_webhook_id,
             )
             if not proceed:
-                session.close()
```

Comment:
These `session.close` calls were removed because they're not doing anything, the Session is a context manager above (see with self.session as session), so it is being closed automatically.